htp/htp_config_private.h

```diff
@@ -238,6 +238,12 @@ struct htp_cfg_t {
     htp_hook_t *hook_request_body_data;
 
     /**
+     * Receives raw request body data, starting immediatelly after empty line
+     * that terminates list of request headers.
+     */
+    htp_hook_t *hook_request_raw_body_data;
```

Comment:
To maintain binary compatibility this should be added to the end of the structure.